src/shipping/components/ShippingZoneRatesPage/ShippingZoneRatesPage.tsx

```diff
@@ -102,6 +108,15 @@ export const ShippingZoneRatesPage: React.FC<ShippingZoneRatesPageProps> = ({
     type: rate?.type || null
   };
 
+  let postalCodesInclusionType = rate?.postalCodeRules[0]?.inclusionType;
+
+  const postalCodeInclusionChange = (
+    inclusion: PostalCodeRuleInclusionTypeEnum
+  ) => {
+    postalCodesInclusionType = inclusion;
```

Comment:
This reassignment has no effect as it will always get called after the initial render and this variable value will always equal to:
`rate?.postalCodeRules[0]?.inclusionType;`